Integrations/integration-ServiceNow.yml

```diff
@@ -1804,6 +1856,8 @@ script:
       defaultValue: normal
     - name: state
       description: State of the ticket, e.g., "Closed" or "7" or "7 - Closed".
+    - name: opened_at
+      description: 'Format: YYYY-MM-DD HH:MM:SS'
```

Comment:
please add the actual description of the argument, and only format
also, please add an example on top of the format